src/client/containers/QuizResultPage/quizResultGetCareer.apiRequest.js

```diff
@@ -0,0 +1,38 @@
+async function getAnswer(id) {
+  const responseFromAnswers = await fetch(`/api/answers/${id}`);
+  const answer = await responseFromAnswers.json();
+  return answer;
+}
+async function careerSet(userId) {
+  let uxPoints = 0;
+  let fullstackPoints = 0;
+  let dataAnalystPoints = 0;
+  let career;
+
+  const responseFromQuizRequest = await fetch(
+    `/api/quiz-results?userId=${userId}`,
+  );
+  const results = await responseFromQuizRequest.json();
+  for (let i = 0; i < results.length; i++) {
+    const answer = await getAnswer(results[i]['fk_answer_id']);
+    uxPoints += Number(answer.ux_points);
+    fullstackPoints += Number(answer.fullstack_points);
+    dataAnalystPoints += Number(answer.dataanalyst_points);
+  }
+  if (uxPoints > fullstackPoints && uxPoints > dataAnalystPoints) {
```

Comment:
Does this work?

Reply:
I have tested it with different values , and got right answers